src/transformers/onnx/features.py

```diff
@@ -63,6 +64,7 @@ class FeaturesManager:
         "bart": supported_features_mapping("default", onnx_config_cls=BartOnnxConfig),
         "mbart": supported_features_mapping("default", onnx_config_cls=MBartOnnxConfig),
         "bert": supported_features_mapping("default", onnx_config_cls=BertOnnxConfig),
+        "ibert": supported_features_mapping("default", onnx_config_cls=IBertOnnxConfig),
```

Comment:
Can you please add all the supported features for IBERT?
Those are:

- masked-lm
- sequence-classification
- ~~multiple-choice~~ (this one is not supported yet)
- token-classification
- question-answering